liquidity/liquidity.go

```diff
@@ -579,23 +655,37 @@ func (m *Manager) dispatchBestEasyAutoloopSwap(ctx context.Context) error {
 
 	// Calculate the amount that we want to loop out. If it exceeds the max
 	// allowed clamp it to max.
-	amount := localTotal - m.params.EasyAutoloopTarget
-	if amount > restrictions.Maximum {
-		amount = restrictions.Maximum
+	amount := localTotal - localTarget
+	satAmount := amount
+
+	// If we run a custom asset, we'll need to convert the asset amount
+	// we want to swap to the satoshi amount.
+	if assetID != "" {
+		satAmount, err = m.cfg.GetAssetPrice(
+			ctx, assetID, assetPeerPubkey, uint64(amount),
+			uint64(restrictions.Minimum),
+		)
+		if err != nil {
+			return err
+		}
+	}
+
+	if satAmount > restrictions.Maximum {
+		satAmount = restrictions.Maximum
 	}
 
 	// If the amount we want to loop out is less than the minimum we can't
 	// proceed with a swap, so we return early.
-	if amount < restrictions.Minimum {
+	if satAmount < restrictions.Minimum {
 		log.Debugf("easy autoloop: swap amount is below minimum swap "+
 			"size, minimum=%v, need to swap %v",
-			restrictions.Minimum, amount)
+			restrictions.Minimum, satAmount)
 		return nil
 	}
 
 	log.Debugf("easy autoloop: local_total=%v, target=%v, "+
 		"attempting to loop out %v", localTotal,
-		m.params.EasyAutoloopTarget, amount)
+		localTarget, amount)
```

Comment:
Just flagging that we should log the `satAmount` here iiuc (though if not renamed, the log is fine).